### PR TITLE
Add rqlite flexibility

### PR DESCRIPTION
**Pull request by tych0**
Hi,

I've been playing around with embedding rqlite into another application. Here's some patches that I found useful while experimenting.

### CODE REVIEW

**Comment by otoolep** [comment]
Thanks very much for your contribution.

File: cmd/rqlite/execute.go [hunk 1]
```diff
@@ -21,9 +21,9 @@ type executeResponse struct {
 }
 
 func execute(ctx *cli.Context, cmd, line string, argv *argT) error {
-	urlStr := fmt.Sprintf("%s://%s:%d/db/execute?pretty&timings", argv.Protocol, argv.Host, argv.Port)
+	urlStr := fmt.Sprintf("%s://%s:%d%sdb/execute?pretty&timings", argv.Protocol, argv.Host, argv.Port, argv.Prefix)
```
Comment (otoolep) [inline cmd/rqlite/execute.go]: What exactly is the point of this change? I don't follow what one would use this switch for.

Comment (tych0) [inline cmd/rqlite/execute.go]: Right now, the client expects the API to live at `/`. I'm embedding rqlite (i.e. not using rqlited, but using the go library) into another application at some level that's not the root, e.g. `/rqlite`. This just allows you to specify something besides `/`.

Comment (otoolep) [inline cmd/rqlite/execute.go]: Got it, thanks @tych0

**Comment by otoolep** [comment]
Thanks @tych0 -- I might tweak the help output slightly, but this does seem useful, particularly the TLS support.

**Comment by tych0** [comment]
No problem, thanks for the quick response!

**Pull request merged**